src/main/kotlin/no/nav/syfo/handlestatus/HandleStatusInvalid.kt

```diff
@@ -42,14 +41,16 @@ suspend fun handleStatusINVALID(
     sha256String: String,
 ) {
 
-    journalService.onJournalRequest(
-        receivedDialogmelding,
-        validationResult,
-        vedleggListe,
-        loggingMeta,
-        pasientNavn,
-        navnSignerendeLege
-    )
+    if (receivedDialogmelding.pasientAktoerId != null) {
```

Comment:
Hvorfor ikke journalføre uten akørid?
Man skal ikke trenge aktørId for å journalføre, i journalføring holder det med personIdent(fnr).

Reply:
Oppretting av journalpost feilet: Server error(https://dokarkiv.intern.nav.no/rest/journalpostapi/v1/journalpost?forsoekFerdigstill=true: 500 Internal Server Error. Text: "{"timestamp":"2022-01-10T08:14:19.827+00:00","status":500,"error":"Internal Server Error","message":"Fant ikke aktørid for person i pdl."

Reply:
I utgangspunktet så burde jo alle FNR ha en tilhørende AktørId. Om vi finner personen hvor vi ikke finner denne koblingen, så burde disse undersøkes nærmere eller kanskje "forkastes" om det betyr at det ikke er en ekte person det gjelder.

Reply:
Det er ikke akkurat noe som skjer ofte: En forekomst i dag og en forekomst 19. november i loggene. Feilmeldingen fra aktoerregisteret er "Den angitte personidenten finnes ikke".

Reply:
Skjønner 👍